index.src.html

```diff
@@ -70,6 +70,9 @@
       text: Navigator
 spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
+    text: same-origin; url: concept-origin-effective-domain
+    text: origin; url: concept-origin
```

Comment:
Right. There are a million `origin` definitions... I'd suggest either following Bikeshed's error advice and adding a `<pre class="link-defaults">` section specifying that you want the one with `for=/` from HTML, but this is fine I guess.